index.bs

```diff
@@ -371,6 +374,28 @@ To <dfn abstract-op>initialize a global object's embedder policy from a response
 
 </div>
 
+### Process a navigation fetch ### {#process-a-navigation-fetch}
+<div algorithm="process a navigation fetch">
+For every navigation request for a nested browsing context whose parent's embedder
+policy is "`require-corp`", CORP is checked.
+
+Run the following step before the sixth step of [=process a navigate fetch=].
```

Comment:
Note that this runs before `frame-ancestors` checks, which sounds fine. I don't think there's any good reason to prefer one ordering to the other, but it's something we should probably lock down in tests so that the right set of things get reported consistently across browsers.

Reply:
Ack.